.github/workflows/updater.yml

```
@@ -67,6 +67,22 @@ on:
     secrets:
       api_token:
         required: true
+    outputs:
+      url:

```

`url` is a bit too generic, maybe `prUrl`, which is in line with `prBranch` suggested below.
```suggestion
      prUrl:
```